release/pkg/assets/config/bundle_release.go

```diff
@@ -50,6 +50,9 @@ var bundleReleaseAssetsConfigMap = []assettypes.AssetConfig{
 					ReleaseImageTagFormat:       "v<eksDReleaseChannel>-<eksDReleaseNumber>",
 				},
 			},
+			{
+				RepoName: "bottlerocket-bootstrap-snow",
```

Comment:
Let's add the ImageTagConfiguration from above here as well, since it's [still getting the non-existent tag](https://github.com/aws/eks-anywhere/pull/4192/files#diff-eb1b40612d15f7d2bb6d4e2ab112bcf9e6d08cca6e3d9bcc7ca9adfce61ca275R483) without it.